src/main/java/me/modmuss50/optifabric/mod/OptifabricSetup.java

```diff
@@ -178,7 +178,10 @@ protected boolean isPresent() {
 
 		Mixins.addConfiguration("optifabric.optifine.mixins.json");
 
-		if (isPresent("cloth-client-events-v0", ">=2.0")) {
+		if (isPresent("fabricloader", ">=0.13.0") && isPresent("cloth-client-events-v0", ">=3.1.61")) {
```

Comment:
this line should be changed to
```java
if (isPresent("fabricloader", ">=0.13.0") && (isPresent("cloth-client-events-v0", ">=3.1.58") || isPresent("cloth-client-events-v0", ">=2.1.60 <3.0") || isPresent("cloth-client-events-v0", ">=1.6.59 <2.0"))) {
```
due to https://github.com/Chocohead/OptiFabric/pull/630#issuecomment-1075535558